cli/cmd/source/source.go

```diff
@@ -5,9 +5,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// sourceCmd represents the source command.
-func SourceCmd(ver string) *cobra.Command {
-	sourceCmd := &cobra.Command{
+// sourceCmd represents the source command
+func SourceCmd(ver version.Version) *cobra.Command {
+	var sourceCmd = &cobra.Command{
```

Comment:
Shouldn't this use `:=`?

Reply:
yeah this was reverted because i disabled `gofumpt` linter and reverted the changes, this will fix automatically after adding this linter again :)